nucypher/cli/actions/select.py

```diff
@@ -94,11 +94,6 @@ def select_client_account(emitter,
     rows = list()
     for index, account in enumerated_accounts.items():
         row = [account]
-        if show_staking:
-            staker = Staker(domain=network, checksum_address=account, registry=registry)
-            staker.refresh_stakes()
-            is_staking = 'Yes' if bool(staker.stakes) else 'No'
-            row.append(is_staking)
```

Comment:
Since you are removing this column of data, you need to also remove L87 & L88 to not specify the column header.

Reply:
Done